core/controllers/domain_objects_validator.py

```diff
@@ -239,6 +240,20 @@ def validate_aggregated_stats(aggregated_stats):
     return aggregated_stats
 
 
+def validate_question_change(obj):
+    """Validates question change.
+
+    Args:
+        obj: dict. Data that needs to be validated.
+
+    Returns:
+        QuestionChange. Returns an QuestionChange object.
+    """
+    # No explicit call to validate_dict method is necessary, because
+    # QuestionChange calls validate method while initialization.
+    return question_domain.QuestionChange(obj)
+
+
```

Comment:
Can you please introduce `from_dict` for the `BaseChange` (it will just call `__init__`) and then in the schema use `object_class` instead of `validation_method`? Also maybe change it in other places where we use some change object.